schema.sql

```diff
@@ -6,5 +6,19 @@ CREATE TABLE animals(
     escape_attempts integer,
     neutered boolean,
     weight_kg decimal,
-    species TEXT
-);
\ No newline at end of file
+    FOREIGN KEY(species_id) REFERENCES species(id),
+    FOREIGN KEY(owner_id) REFERENCES species(id)
+);
+
+CREATE TABLE owners(
+    id SERIAL PRIMARY KEY,
+    full_name text,
+    age integer
+);
+
+CREATE TABLE species(
+    id SERIAL PRIMARY KEY,
+    name text
```

Comment:
- [ ] Kindly, do the same for this column too. change the `text` data type from the name table to `varchar(n)` data type.